### DISCUSSION TITLE
Praeco for simple elastalert management

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#12177] [Ideas]
would be nice to integrate praeco https://github.com/johnsusek/praeco into SecurityOnion for simple elastalert management.

**Comment by reyesj2** [comment]
During our 2023 Security Onion Conference an announcement was made regarding a new SOC interface named Detections. Check it out here https://youtu.be/aUshk9lLuA4?si=przfb8catJWNb9E9&t=2042
Once this is released you can review the update and let us know if it meets the requirements you have
